capsules/src/rf233.rs

```diff
@@ -799,48 +891,85 @@ impl<'a, S: spi::SpiMasterDevice + 'a> radio::Radio for RF233<'a, S> {
     fn set_transmit_client(&self, client: &'static radio::TxClient) {
         self.tx_client.set(Some(client));
     }
+
     fn set_receive_client(&self, client: &'static radio::RxClient, buffer: &'static mut [u8]) {
         self.rx_client.set(Some(client));
         self.rx_buf.replace(buffer);
     }
 
+    fn set_config_client(&self, client: &'static radio::ConfigClient) {
```

Comment:
Yeah, sure they _could_ be, but that's implementable in theory on top of a single client trait. You could, e.g., say the same thing for the UART client (may want want for reading and one for transmitting), but that seems pretty rare. My question is whether we anticipate this being common (in which case it makes sense to have two separate clients) or rare (in which case they ought to be the same client).